vars/capsuleSystemTests.groovy

```diff
@@ -183,12 +183,25 @@ void call(Map additionalConfig) {
           SYSTEM_TESTS_LOG_OUTPUT="\"${testNetworkDir}/log-output\""
         }
         steps {
+          // need to wrap TEST_DIRECTORY and split it by space if multiple directories are passed as current makefile does not support passing something like 'tests/API tests/LNL' in one throw
           dir('system-tests/scripts') {
-              sh 'make test'
+            script {
+              if ("${params.TEST_DIRECTORY}".split(' ').size() > 1){
+                "${params.TEST_DIRECTORY}".split(' ').each { dir ->
+                  withEnv([
+                    "TEST_DIRECTORY=${dir}"
+                  ]) {
+                    sh 'make test'
+                  }
+                }
+              }
+              else {
+                sh 'make test'
+              }
+            }
```

Comment:
Please note that currently:
```bash
TEST_DIRECTORY= "\"${params.SYSTEM_TESTS_TEST_DIRECTORY}\""
```
so the tests are run with 
```bash
pytest ... "tests/sub-dir-1 tests/sub-dir-2"
```
pytest treats the whole string `"tests/sub-dir-1 tests/sub-dir-2"` as a single directory.

Please remove the wrapping, and it should work:
```diff
- TEST_DIRECTORY= "\"${params.SYSTEM_TESTS_TEST_DIRECTORY}\""
+ TEST_DIRECTORY= "${params.SYSTEM_TESTS_TEST_DIRECTORY}"
```